File: messaging/v1/alpha_sender/fetch-service-alpha-get/fetch-service-alpha-get.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Messaging.V1.Service;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var alphaSender = AlphaSenderResource.Fetch(
            pathServiceSid: "MG2172dd2db502e20dd981ef0d67850e1a",
            pathSid: "AIc781610ec0b3400c9e0cab8e757da937"
        );

        Console.WriteLine(alphaSender.Sid);
    }
}

File: messaging/v1/alpha_sender/fetch-service-alpha-get/fetch-service-alpha-get.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

alpha_sender = @client.messaging
                      .services('MG2172dd2db502e20dd981ef0d67850e1a')
                      .alpha_senders('AIc781610ec0b3400c9e0cab8e757da937')
                      .fetch

puts alpha_sender.sid
